jstests/telemetry/telemetry_single_node.js

```diff
@@ -0,0 +1,62 @@
+(function() {
+"use strict";
+load('jstests/telemetry/_telemetry_helpers.js');
+
+var telmTestSingle = function() {
+    mkdir(telmPath);
+    cleanupTelmDir();
+
+    var singleTest = MongoRunner.runMongod({
+        setParameter: setParameterOpts
+    });
+
+    //test perconaTelemetryGracePeriod
+    sleep(3000);
+    var telmFileList = listFiles(telmPath);
+    assert.eq(1,telmFileList.length,telmFileList);
```

Comment:
@olexandr-havryliak  This seems to be incorrect way of testing grace period. Let's assume the grace period doesn't work at all and the telemetry is scraped for the first time immediately after mongod startup. Providing that the scrape period is 5 seconds in `setParameterOpts`, we will still get one file after 3 second sleep.
The proper way would be to sleep for `grace_period - 1` seconds, check that there are _no_ files in `telmPath`, then sleep for 2 more seconds and check there is one file now.

Reply:
the fix is added to this PR